### PR TITLE
reset rhsso imagestream to 7.2

### PR DESCRIPTION
**Pull request by philbrookes**
## Verification (INTLY 1347)
- run the install, see that rhsso is running 7.2

## Verification (INTLY-741)
- (existing clusters) run uninstall and `rm -rf /tmp/*`
- run the install
- login to rhsso 
- load customer-admin user 
- go to role bindings tab 
- in client roles select realm-management 
- check that manage-users is assigned

### CODE REVIEW

**Comment by maleck13** [comment]
Will test on my cluster

**Comment by philbrookes** [comment]
@maleck13 any progress on your verification?

**Comment by maleck13** [comment]
installing

**Comment by aidenkeating** [comment]
:eyes:

**Comment by philbrookes** [comment]
@maleck13 @aidenkeating I'm pretty confident this is working now, if you have time to verify again, I'd appreciate it!

**Comment by aidenkeating** [comment]
:eyes:

**Comment by aidenkeating** [comment]
## SSO SSO
Screen Shot 2019-03-22 at 09 28 58
## Launcher SSO
Screen Shot 2019-03-22 at 09 28 30
## Customer Admin Roles
Screen Shot 2019-03-22 at 09 22 54

File: inventories/group_vars/all/manifest.yaml [hunk 1]
```diff
@@ -49,15 +49,17 @@ launcher_backend_image_tag: '7a7fd07'
...
 # rhsso is not an optional component
-rhsso_version: '7.3.0.GA'
+rhsso_version: '7.2.0.GA'
 #below vars not currently used but will be used to pull in the new versions of RH-SSO once we decouple the resources from the installer.
-rhsso_imagestream_name: redhat-sso73-openshift:1.0
-rhsso_imagestream_image: registry.access.redhat.com/redhat-sso-7/sso73-openshift:1.0
-rhsso_operator_release_tag: 'v1.1.0'
+rhsso_imagestream_name: redhat-sso72-openshift:1.4
+rhsso_imagestream_image: registry.access.redhat.com/redhat-sso-7/sso72-openshift:1.4
+rhsso_operator_release_tag: 'master'
```
Comment (mikenairn) [inline inventories/group_vars/all/manifest.yaml]: @philbrookes @maleck13 Was this intentional? We should i would have thought had this set to an explicit version like everything else?

Comment (maleck13) [reply inventories/group_vars/all/manifest.yaml]: this should be explicit

**Pull request merged**